l10n_ve_withholding/model/invoice.py

```diff
@@ -49,8 +49,8 @@ def test_retenida(self, cr, uid, ids, *args):
                    'from account_move_line l '
                    'inner join account_journal j on (j.id=l.journal_id)'
                    ' where l.id in (' + ','.join(
-                       [str(item) for item in res]) + ') and j.type=' + '\''
-                   + type_journal + '\'')
+                       [str(item) for item in res]) + ') and j.type=' +
+                   '\'' + type_journal + '\'')
```

Comment:
Can you please take the SQL Sentence away from the method and create it as a Variable outside the class,
then call the variable inside the cr.execute and pass the parameters.